lib/templates/block_template.mustache

```diff
@@ -151,7 +151,7 @@
     window._pxUuid = '{{uuid}}';
     window._pxHostUrl = '{{{hostUrl}}}';
 </script>
-<script> {{{blockScript}}}</script>
+<script src="{{{blockScript}}}"></script>
```

Comment:
not related to this change, but `window._pxFirstPartyEnabled = '{{firstPartyEnabled}}';` should be changed to `window._pxFirstPartyEnabled = {{firstPartyEnabled}};`

Reply:
done